src/MuTalk-Utilities-Tests/MTMutantOperatorAnalysisTest.class.st

```diff
@@ -0,0 +1,59 @@
+Class {
+	#name : 'MTMutantOperatorAnalysisTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'operatorAnalysis'
+	],
+	#category : 'MuTalk-Utilities-Tests',
+	#package : 'MuTalk-Utilities-Tests'
+}
+
+{ #category : 'running' }
+MTMutantOperatorAnalysisTest >> setUp [
+
+	super setUp.
+
+	operatorAnalysis := MTMutantOperatorAnalysis forClasses:
+		                    { MTAuxiliarClassForMutantOperatorAnalysis }
+]
+
+{ #category : 'tests' }
+MTMutantOperatorAnalysisTest >> testGetAllOperators [
+
+	| actual expected |
+	actual := (operatorAnalysis operatorsProducingOverXMutants: 0) keys
```

Comment:
the name of this method is strange :)

`operatorsProducingOverXMutants:`

Does it mean they produce more than 0 mutants?
Alsy, why is it answering a dictionary? I see all your tests do `keys asSet`, `keys asSet`, `keys asSet`. This means that probably we want a method returning the set of keys directly and hide the fact that internally it's a dictionary right? :D

Reply:
Actually, `Over 0` means `> 0` or `>= 0`? Maybe a little rename could clarify "At least" vs "More than"

Reply:
I think it can be useful to return a dictionary, to actually see how many mutants are produced by each operator.
But yes I think I could add a bit more methods, like the whole dictionary, the set of mutants producing at least a certain number of mutants, the 10 most producing operators, things like that.

Reply:
Actually there is already a method returning the whole dictionary, forgot about it ahah